src/core/types/addons.js

```diff
@@ -18,6 +18,7 @@ export type AddonVersionType = {|
 export type AddonAuthorType = {|
   name: string,
   url: string,
+  username: string,
```

Comment:
Hmm, this field isn't documented. Can you file a bug to update the docs? https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#detail

Reply:
It seems to be documented here https://github.com/mozilla/addons-server/pull/6385/files#diff-d054974b06ee8bc0688e68b6331256ec but not deployed yet.

Reply:
I think RTD is having trouble updating. I'm checking on it...